pylmod/tests/test_gradebook.py

```diff
@@ -574,6 +574,55 @@ def test_spreadsheet2gradebook_multi(self):
                  u'studentId': None},
             ])
         )
+        # Verify that we got the same grades, setting auto-approve = False
+        gradebook._spreadsheet2gradebook_multi(
+            csv_reader=spreadsheet,
+            email_field='External email',
+            non_assignment_fields=['External email'],
+            approve_grades=False
+        )
+        # Verify that we got the grades we expect
+        last_request = httpretty.last_request()
+        self.assertEqual(
+            last_request.body,
+            json.dumps([
```

Comment:
This structure is now being used 3 times, you should assign it to a variable and reuse in all three of these tests